### PR TITLE
Openstack rollover

### PR DESCRIPTION
**Pull request by pierreprinetti**
This change rationalises the frequencies of OpenStack periodics. Old
versions get fewer runs, whereas the latest upgrade gets double the
frequency.

In particular:

* Lower the frequency of old versions periodics
* Lower the frequency of old versions upgrade periodics
* Lower the frequency of techpreview runs
* Remove 'serial' runs of old versions periodics
* Raise the frequency of the latest version upgrade periodic

In a separate commit, remove the step-registry based upgrade test 4.5->4.6, because installing 4.5 is not supported by our step-registry jobs and the run is therefore doomed to permafail.

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.6.yaml [hunk 1]
```diff
@@ -93,14 +93,14 @@ tests:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-ovn
 - as: e2e-openstack-parallel
-  cron: 0 0,8,16 * * *
+  interval: 72h
```
Comment (mandre) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.6.yaml:96]: Make it 48h, to be consistent with the serial job and other platforms.

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by pierreprinetti** [comment]
/assign @petr-muller
as per the bot suggestion

**Review by petr-muller — APPROVED** [review]

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**